boost_manager/src/settings.rs

```diff
@@ -9,6 +9,9 @@ pub struct Settings {
     /// "poc_entropy=debug,poc_store=info"
     #[serde(default = "default_log")]
     pub log: String,
+    /// File name to be watched by custom tracing
+    #[serde(default = "default_tracing_cfg_file")]
+    pub tracing_cfg_file: String,
```

Comment:
Can we follow the poc-metrics and file-store pattern of using a libs settings struct.
```suggestion
    pub custom_tracing: custom_tracing::Settings,
```

Reply:
Yeah I was thinking of doing that. 👍🏻